edb/lib/std/30-jsonfuncs.edgeql

```diff
@@ -241,8 +241,12 @@ CREATE CAST FROM std::json TO anytuple {
 CREATE CAST FROM std::json TO array<json> {
     SET volatility := 'Immutable';
     USING SQL $$
-        SELECT array_agg(j)
-        FROM jsonb_array_elements(val) AS j
+    SELECT CASE jsonb_typeof(val)
```

Comment:
[`NULLIF`](https://www.postgresql.org/docs/current/functions-conditional.html#FUNCTIONS-NULLIF) would probably be faster here.

Reply:
I'm missing the obvious way to to use NULLIF here?

Reply:
Try

```
SELECT array_agg(j) 
FROM jsonb_array_elements(nullif(val, 'null'::jsonb)) as j;
```

Reply:
Oh, aha